lib/test_sites/cac.rb

```diff
@@ -8,23 +8,33 @@
 module TestSites
   MILES_PER_METER = (1 / 1609.344)
   # Utility class for processing Coders Against Covid location data.
+  # rubocop:disable Metrics/ClassLength
   class CAC
     SEPARATOR = ', '
 
+    def self.cac_data
+      CAC.new.all_data
+    end
+
     def self.all_hours
-      CAC.cac_data.map { |cac_entry| cac_entry['location_hours_of_operation'] }
+      CAC.new.all_hours
     end
 
-    def self.cac_data
-      TestSites::CACClient.new.locations.filter_map do |location|
-        next if location.deleted_on.present?
+    def all_hours
```

Comment:
@ivanoblomov - I did this because I needed to call `cac_data` and `all_hours` within the same method, and with both as class methods there was no way to avoid a redundant API request.